ui/lib/pki/addon/routes/issuers/index.js

```diff
@@ -0,0 +1,23 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PkiIssuersIndexRoute extends Route {
+  @service store;
+  @service secretMountPath;
+  @service pathHelp;
+
+  async model() {
+    // the pathHelp service is needed for adding openAPI to the model
+    await this.pathHelp.getNewModel('pki/pki-issuer-engine', 'pki');
+
+    return await this.store
```

Comment:
One small nit, you can return a promise from the model hook so you don't need the `await` here.

Reply:
I always forget that. Thank you!